pallets/asset/src/lib.rs

```diff
@@ -1374,6 +1367,22 @@ impl<T: Trait> AssetSubTrait<T::Balance> for Module<T> {
 /// Public functions can be called from other modules e.g.: lock and unlock (being called from the tcr module)
 /// All functions in the impl module section are not part of public interface because they are not part of the Call enum.
 impl<T: Trait> Module<T> {
+    /// Ensure that all `idents` are valid.
+    fn ensure_asset_idents_valid(idents: &[AssetIdentifier]) -> DispatchResult {
+        for ai in idents {
+            ensure!(ai.is_valid(), Error::<T>::InvalidAssetIdentifier);
+        }
```

Comment:
This is from my branch. NOT mandatory, use it if you think is more readable (Do not apply directly).
```suggestion
        ensure!( idents.iter().all(AssetIdentifier::is_valid).is_some(), Error::<T>::InvalidAssetIdentifier);
```

Reply:
Yes definitely better 👍, will apply, thanks :)